components/organisms/notification/NotificationItem.tsx

```diff
@@ -0,0 +1,72 @@
+import { Span } from '@/components/atoms';
+import { useCalcRegisterDate } from '@/hooks/index';
+import { NOTIFICATION_LIST } from '@/constants/index';
+import styled from 'styled-components';
+import { Layout } from '@/styles/theme';
+import { useRouter } from 'next/router';
+
+const Container = styled.li`
+  ${Layout.flexRowStartEnd}
+  padding: 16px;
+  box-sizing: border-box;
+  border-bottom: ${({ theme }) => `1px solid ${theme.GRAY02}`};
+  height: 70px;
+  cursor: pointer;
+`;
+
+const TypeContainer = styled.div`
+  margin-right: 8px;
+`;
+const ContentContainer = styled.div`
+  width: 100%;
+  overflow: hidden;
+`;
+const Title = styled.p`
+  font-size: 16px;
+  margin: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+`;
+
+const Description = styled.div`
+  ${Layout.flexRowBetweenEnd}
+  font-size: 12px;
+  margin-bottom: 4px;
+`;
+
+const StyledSpan = styled(Span)`
+  color: ${({ theme }) => theme.GRAY05};
+  font-size: 12px;
+  line-height: 1.1;
+`;
+
+const NotificationItem = ({ notification }: any) => {
```

Comment:
타입 정의를 해놓으셨으면 쓰는게 좋을거같습니다!
```suggestion
const NotificationItem = ({ notification }: { notification: INotification }) => {
```